src/pydata_sphinx_theme/__init__.py

```diff
@@ -1,65 +1,37 @@
-"""
-Bootstrap-based sphinx theme from the PyData community
-"""
+"""Bootstrap-based sphinx theme from the PyData community."""
+
+import json
 import os
 from pathlib import Path
-from functools import lru_cache
-import json
-from urllib.parse import urlparse, urlunparse
-import types
+from typing import Dict
+from urllib.parse import urlparse
 
-import jinja2
-from bs4 import BeautifulSoup as bs
-from docutils import nodes
-from sphinx import addnodes
-from sphinx.application import Sphinx
-from sphinx.environment.adapters.toctree import TocTree
-from sphinx.addnodes import toctree as toctree_node
-from sphinx.transforms.post_transforms import SphinxPostTransform
-from sphinx.util.nodes import NodeMatcher
-from sphinx.errors import ExtensionError
-from sphinx.util import logging, isurl
-from sphinx.util.fileutil import copy_asset_file
-from pygments.formatters import HtmlFormatter
-from pygments.styles import get_all_styles
 import requests
 from requests.exceptions import ConnectionError, HTTPError, RetryError
+from sphinx.application import Sphinx
+from sphinx.errors import ExtensionError
+from sphinx.util import logging
 
-from .translator import BootstrapHTML5TranslatorMixin
+from .edit_this_page import setup_edit_url
+from .logo import copy_logo_images, setup_logo_path
+from .pygment import overwrite_pygments_css
+from .short_link import ShortenLinkTransform
+from .toctree import add_toctree_functions
+from .translator import setup_translators
+from .utils import config_provided_by_user, get_theme_options
```

Comment:
WDYT about putting these in a directory with an `__init__`?  So this would become, e.g., 

```py
from ._helpers import setup_edit_url, copy_logo_images, setup_logo_path, etc
```

IDK what the best name is (probably doesn't matter much). Ideas: `_tools`, `_pst`, `_helpers`, `_init_helpers`, etc

Reply:
I'm not a big fan of creating a extra folder to store them but I see you point of reducing the number of import lines. Let me try something

Reply:
what do you think of the modifications made in https://github.com/pydata/pydata-sphinx-theme/pull/1271/commits/71f73c25412afb63dcc3021635242f83062aee93 ? It avoids the extra folder and greatly simplify the imort statements.

Reply:
good enough for me. Personally I prefer having in a folder (when I `ls` a directory shorter lists are easier for me to scan quickly) but it's a matter of taste so I won't push the issue.